docs/sphinx/developers/cpp/maintenance.txt

```diff
@@ -0,0 +1,238 @@
+         <Link>
+           <AdditionalOptions>%(AdditionalOptions)</AdditionalOptions>
+    -      <AdditionalDependencies>ws2_32.lib;advapi32.lib;icuuc.lib;%(AdditionalDependencies)</AdditionalDependencies>
+    +      <AdditionalDependencies>ws2_32.lib;advapi32.lib;icuucd.lib;%(AdditionalDependencies)</AdditionalDependencies>
+           <OutputFile>$(TargetPath)</OutputFile>
+           <AdditionalLibraryDirectories>%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
+           <GenerateDebugInformation>true</GenerateDebugInformation>
+    @@ -578,7 +578,7 @@
+           <Culture>0x0409</Culture>
+         </ResourceCompile>
+         <Link>
+    -      <AdditionalDependencies>ws2_32.lib;advapi32.lib;%(AdditionalDependencies)</AdditionalDependencies>
+    +      <AdditionalDependencies>ws2_32.lib;advapi32.lib;icuucd.lib;%(AdditionalDependencies)</AdditionalDependencies>
+           <OutputFile>$(TargetPath)</OutputFile>
+           <AdditionalLibraryDirectories>%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
+           <GenerateDebugInformation>true</GenerateDebugInformation>
+    @@ -669,7 +669,7 @@
+           <Culture>0x0409</Culture>
+         </ResourceCompile>
+         <Link>
+    -      <AdditionalDependencies>ws2_32.lib;advapi32.lib;%(AdditionalDependencies)</AdditionalDependencies>
+    +      <AdditionalDependencies>ws2_32.lib;advapi32.lib;icuuc.lib;%(AdditionalDependencies)</AdditionalDependencies>
+           <OutputFile>$(TargetPath)</OutputFile>
+           <AdditionalLibraryDirectories>%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
+           <GenerateDebugInformation>true</GenerateDebugInformation>
+
+- Check for a supported set of project files under
+  :file:`projects/Win32/VC<ver>`.  If a suitable version does not
+  exist:
+
+  - copy :file:`projects/Win32/VC12` (the latest at the time of
+    writing) to :file:`projects/Win32/VC<ver>`; make sure you applied
+    the patch in the previous step beforehand
+  - Start the Visual Studio application
+  - Open the :file:`projects/Win32/VC<ver>/xerces-all/xerces-all.sln`,
+    and allow Visual Studio to upgrade the projects to the current
+    version
+  - Save the solution (you must Save As to overwrite the existing
+    solution; Save is insufficient)
+    
+- Copy the solution and project files under
+  :file:`packages/xerces/files/projects/Win32/VC<ver>`, replacing the
+  existing contents
+
+
+Toolchain support and standards conformance
+-------------------------------------------
+
+Currently the code is built and tested on several platforms,
+including:
+
+- FreeBSD with clang++ 3.4
+- Linux with GCC 4.8 and 4.9
+- MacOS X with clang++ 3.4 and 3.6 (non-standard Mac versions)
+- Windows with VC12
+
+Testing on a range of version combinations of compilers and platforms
+is helpful in picking up bugs which would otherwise go undiagnosed
+until encountered by an end user.  For example, the FreeBSD builds
```

Comment:
when talking about boost, "end-user" was hyphenated, maybe didn't need to be if it isn't here -- at least, seems inconsistent